input/ig.xml

```diff
@@ -30,9 +30,9 @@
 	<packageId value="hl7.fhir.au.base"/>
 	<license value="CC0-1.0"/>
 	<fhirVersion value="4.0.1"/>
-		<dependsOn>
+	<dependsOn>
```

Comment:
dependency for the extension is not required explicitly. It comes in automatically now like terminololgy.

Reply:
Done